Zend/zend_portability.h

```diff
@@ -864,4 +865,21 @@ static zend_always_inline uint64_t ZEND_BYTES_SWAP64(uint64_t u)
 # define ZEND_OPCACHE_SHM_REATTACHMENT 1
 #endif
 
+#ifndef CHAR_BITS
+# define CHAR_BITS 8
+#endif
+
+#ifndef UINTPTR_WIDTH
+# define UINTPTR_WIDTH (CHAR_BITS * sizeof(uintptr_t))
```

Comment:
Very nitpicky: Note that sizeof is always expressed in CHAR_BITS so the multiplication is not logical (e.g. sizeof(char) is _always_ 1)